tcpdf.php

```diff
@@ -9925,7 +9925,7 @@ protected function _putcatalog() {
 				}
 				$out .= ' >> >>';
 			}
-			$font = $this->getFontBuffer('helvetica');
+			$font = $this->getFontBuffer((($this->pdfa_mode) ? 'pdfa' : '') .'helvetica');
```

Comment:
So there is a pdfahelvetica mode?

Can you create a new example in the examples folder?
Or update one that is for pdfa?

This will be run as a test on our test suite

Reply:
the sample 065 just needs a 

```php
$certificate = 'file:///my.pem';
$privateKey  = 'file://my.prv.key.pem';
// set document signature
$pdf->setSignature($certificate, $privateKey, '', '', 1, array());
```
for the error to pop up